requirements-dev.txt

```diff
@@ -7,3 +7,4 @@ flake8==3.9.2
 isort==5.9.1
 pre-commit==2.13.0
 pytest-django==4.4.0
+web3==5.20.1
```

Comment:
Moved the `web3` dependency to the `requirements-dev.txt` as it is only being used directly with the unit tests (to generate addresses)

Reply:
I like this, but gnosis-py requires web3 and will install it, so no difference 😉